packages/api/src/modules/association-name/repositories/associationName.repository.ts

```diff
@@ -96,7 +96,7 @@ export class AssociationNameRepository extends MongoRepository<AssociationNameEn
     }
 
     async createIndexes() {
-        await this.collection.createIndex({ siren: 1, rna: 1, name: 1 }, { unique: true });
+        // await this.collection.createIndex({ siren: 1, rna: 1, name: 1 }, { unique: true });
```

Comment:
Peut-être rajouter un commentaire genre "Will be handled in #XXX" ? Au cas où ces travaux traînent